<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_activite</class>
 <widget class="QDialog" name="Dialog_activite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>691</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_ajouter">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_Code_act">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Type_act">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Libelle_act">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_DateH_act">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CODE_ACTIVITE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TYPE_ACTIVITE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>LIBELLE_ACTIVITE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DATE_H</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ajouter">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>270</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_supprimer">
    <attribute name="title">
     <string>Supprimer</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Tapez le code de la activite que vous souhaite modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>260</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ide_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>100</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_modifier">
    <attribute name="title">
     <string>Modifier</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_modifier">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>260</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_code_mod">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_type_mod">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_libelle_mod">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_dateH_mod">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>40</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tapez le code que vous souhaite le modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Type Activite</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Libelle Activite</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date_H</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_consulter">
    <attribute name="title">
     <string>Consulter</string>
    </attribute>
    <widget class="QTableView" name="tableactivite">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>441</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Trier">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>290</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Trier</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_chercher">
    <attribute name="title">
     <string>Chercher</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_chercher">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>290</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Chercher</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableactivite_chercher">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>541</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_identifiant">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>290</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Vous devez entrer le code de l'activite</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Chercher par date</string>
    </attribute>
    <widget class="QComboBox" name="comboBox_activite">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_code_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_type_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_libelle_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>180</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_dateH_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Code Activite</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Type Activite</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Libelle Activite</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date_H</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
